server/bin/fxa-content-server.js

```diff
@@ -140,6 +144,9 @@ function makeApp() {
   // it's a four-oh-four not found.
   app.use(fourOhFour);
 
+  // The error handler must be before any other error middleware
+  app.use(raven.ravenModule.middleware.express.errorHandler(raven.ravenMiddleware));
```

Comment:
~~Should this be above `app.use(fourOhFour);` since that is handling an error?~~ Nope, `fourOhFour` is a regular handler and not error handler.